docs/buildah-add.md

```diff
@@ -1,4 +1,4 @@
-# buildah-add "1" "March 2017" "buildah"
+# buildah-add "7" "April 2021" "buildah"
```

Comment:
should this be 7?  If so, do we have other references in other man pages that need to change?  Ditto same question for copy below.

Reply:
Oh, no, for some reason my brain farted and thought that `1` was the day of the month.  Reverting it.